vita3k/gui/src/home_screen.cpp

```diff
@@ -523,10 +523,11 @@ void select_app(GuiState &gui, const std::string &title_id) {
         LOG_ERROR("App with title id {} not found", title_id);
 }
 
-static constexpr ImU32 ARROW_COLOR = 0xFFFFFFFF; // White
-static float scroll_type, current_scroll_pos, max_scroll_pos;
-
 void draw_home_screen(GuiState &gui, EmuEnvState &emuenv) {
+    static constexpr ImU32 ARROW_COLOR = 0xFFFFFFFF; // White
```

Comment:
```suggestion
    constexpr ImU32 ARROW_COLOR = 0xFFFFFFFF; // White
```
No need to make a constexpr integer static